content/02-molecules.html

```diff
@@ -130,16 +130,35 @@ <h2 class=st-title>Social form validate</h2>
       
       <ul class=socials-form>
         <li class="icon email valid">
-          <input name=email placeholder=mycontact@email.com type=email value=value>
+          <input name=email placeholder=mycontact@email.com type=email value=email_value>
```

Comment:
I'm wondering if we should actually have values here as it will hide the placeholder…
❓

Reply:
I open an issue for this one and enhance it in a new PR ;)